File: 2.Implement/Delivery_Server/src/main/java/com/delivery/app/Delivery/data/dto/request/cart/SaveCartDto.java
package com.delivery.app.Delivery.data.dto.request.cart;

import com.delivery.app.Delivery.data.entity.Stock;
import lombok.Getter;

import java.util.ArrayList;

@Getter
public class SaveCartDto {
    Long userId;
    Long marketId;
    ArrayList<Long> itemIdList;
    ArrayList<Integer> itemNumList;
    String marketName;
    Integer cost;
}

File: 2.Implement/DeliveryApp/app/src/main/java/com/project/deliveryapp/retrofit/dto/request/cart/SaveCartDto.kt
package com.project.deliveryapp.retrofit.dto.request.cart

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class SaveCartDto(
    val userId: Long,
    val marketId: Long,
    val marketName: String,
    val itemIdList: List<Long>,
    val itemNumList: List<Int>,
    val cost: Int
): Parcelable
